src/syscheckd/run_check.c

```diff
@@ -240,13 +240,23 @@ void start_daemon()
         syscheck.time = 604800;
     }
 
-    // Deleting content local/diff directory
-    char diff_dir[PATH_MAX];
+    // Deleting content of FIM diff directory
+    char diff_file_dir[PATH_MAX];
+    char diff_registry_dir[PATH_MAX];
+    char diff_local_dir[PATH_MAX];
 
-    snprintf(diff_dir, PATH_MAX, "%s/local/", DIFF_DIR);
+    snprintf(diff_file_dir, PATH_MAX, "%s/file/", DIFF_DIR);
+    snprintf(diff_registry_dir, PATH_MAX, "%s/registry/", DIFF_DIR);
+    snprintf(diff_local_dir, PATH_MAX, "%s/local/", DIFF_DIR);
 
-    if (cldir_ex(diff_dir) == -1 && errno != ENOENT) {
-        merror("Unable to clear directory '%s': %s (%d)", diff_dir, strerror(errno), errno);
+    if (cldir_ex(diff_file_dir) == -1 && errno != ENOENT) {
```

Comment:
Why do we need to remove the `file` and `registry` folders now?

Reply:
It is to delete the folders used by report_changes. There has been a name change, now we use the folders file and registry. It is necessary to keep deleting the old folder, called local, in case of upgrades.

Reply:
Done